compiler/src/iree/compiler/Codegen/Common/GPU/GPUHeuristics.cpp

```diff
@@ -94,6 +150,10 @@ static bool isValidMMASchedule(const GPUMatmulShapeType &problem,
   return isAligned && isDistributableLhs && isDistributableRhs;
 }
 
+// Tries to fit the schedule into shared memory by decrementing the size of the
+// schedule dimensions in reverse lexicographical order until a valid schedule
```

Comment:
"reverse lexicographical order" sounds confusing for numbers, no? Can we use ascending/descending?